samples/sample_encode/src/pipeline_region_encode.cpp

```diff
@@ -488,9 +484,11 @@ mfxStatus CRegionEncodingPipeline::ResetMFXComponents(sInputParams* pParams)
     sts = AllocFrames();
     MSDK_CHECK_STATUS(sts, "AllocFrames failed");
 
+    auto hevcRegion = m_mfxEncParams.AddExtBuffer<mfxExtHEVCRegion>();
```

Comment:
**probably** need to move to to the block below. Please also pay attention to comment below

Reply:
Done